handlers/routes.py

```diff
@@ -1038,16 +1041,18 @@ def render_and_send_docs(user, thisdoc, nda_file_base64, google_credentials_info
             notification_subject = "Your Document {} has been downloaded".format(thisdoc.doc_id)
             analytics_link = "{}{}analytics/{}".format(conf.BASE_URL, BASE_PATH, thisdoc.doc_id)
 
-            mymail.send(subject="Documentation", email_from=sender_format.format(user.org_name, conf.SMTP_EMAIL),
+            mymail.send(subject=thisdoc.wp_name, email_from=sender_format.format(user.org_name, conf.SMTP_EMAIL),
                         emails_to=[signer_user.email],
                         attachments_list=attachments_list,
-                        html_message=DEFAULT_HTML_TEXT + button.generate().decode("utf-8"))
+                        html_message=email_body_html + button.generate().decode("utf-8"),
```

Comment:
El html ya viene desde el cliente?

Reply:
SI si me parece que puede haber un hueco de seguridad por dejar recibir un html foraneo, bueno aunque si es interno nada mas, creo que con poner algun TODO comment como 
`# TODO check security injection on this field `

Reply:
si, en este caso viene de prescrypto en un post, ese html se reemplaza en el body del email